$OpenBSD: patch-build_src_glbuild_c,v 1.1.1.1 2012/07/17 21:29:50 rfreeman Exp $

allow dlopen to find our libGL.so, falling back to libGL.so.1 for
linux or others.

--- build/src/glbuild.c.orig	Tue Oct 11 17:52:53 2011
+++ build/src/glbuild.c	Fri Mar  9 01:12:27 2012
@@ -338,6 +338,8 @@ int32_t loadgldriver(const char *driver)
         driver = "opengl32.dll";
 #elif defined __APPLE__
         driver = "/System/Library/Frameworks/OpenGL.framework/OpenGL";
+#elif defined __OpenBSD__
+	driver = "libGL.so";
 #else
         driver = "libGL.so.1";
 #endif
@@ -936,6 +938,8 @@ int32_t loadglulibrary(const char *driver)
         driver = "glu32.dll";
 #elif defined __APPLE__
         driver = "/System/Library/Frameworks/OpenGL.framework/OpenGL"; // FIXME: like I know anything about Apple.  Hah.
+#elif defined __OpenBSD__
+        driver = "libGLU.so";
 #else
         driver = "libGLU.so.1";
 #endif
